### ISSUE TITLE
Some problems with using other data sets for speech enhancement

### ISSUE THREAD

**Issue by Huer-H** [deepakbaby/se_relativisticgan#10]
Hi ！
Thank you for your work. I reproduce your experiments, and also achieved good results.
However, when I use my own data set, the PESQ indicator for enhanced speech is much lower than the PESQ indicator for noisy speech.
Can you give me some suggestions?
The noise I used was Nonspeech and Noisex-92
Noisy speech synthesis is a code written by myself.

**Comment by deepakbaby** [comment]
Do your dataset has a training set? Or is it just that you trained the speech enhancement system using the database used in this repo and simply fed your test data for enhancement?

**Comment by Huer-H** [comment]
Using my data to retrain, the network parameters have not been modified.
In the enhancement phase, the test set is used for speech enhancement. The enhanced speech has a poor PESQ index, which is not as good as the PESQ value of unprocessed noisy speech.

**Comment by deepakbaby** [comment]
I assume that training with your new training data went well with the losses reaching equilibrium (all losses ~0.69). If this is not the case, it means that your GAN training was unstable.
If you indeed trained it properly, then it would be better to plot the waveforms, listen to them and see where the difference is coming from.

**Comment by YeBobr** [comment]
> Hi ！
> Thank you for your work. I reproduce your experiments, and also achieved good results.
> However, when I use my own data set, the PESQ indicator for enhanced speech is much lower than the PESQ indicator for noisy speech.
> Can you give me some suggestions?
> The noise I used was Nonspeech and Noisex-92
> Noisy speech synthesis is a code written by myself.

Hello, 
Do you change the hyper-parameters in opts? Why do I reproduce the results in the paper? I ommit the instance normalization in discriminator according to the original paper. 
Thanks a lot !
